tests/e2e-cypress/integration/drawing/kml.cy.js

```diff
@@ -24,6 +25,38 @@ describe('Drawing new KML', () => {
             cy.checkKMLRequest(interception, [EditableFeatureTypes.ANNOTATION])
         )
     })
+    it('keep KML after a page reload', () => {
+        cy.goToDrawing()
+        cy.clickDrawingTool(EditableFeatureTypes.ANNOTATION)
+        cy.get(olSelector).click('center')
+        cy.wait('@post-kml').then((interception) =>
+            cy.checkKMLRequest(interception, [EditableFeatureTypes.ANNOTATION])
+        )
+        cy.reload(true)
+        cy.waitUntilState((state) => {
+            return state.layers.activeLayers.length > 0
+        })
+        cy.readStoreValue('state.layers.activeLayers').then((activeLayers) => {
+            expect(activeLayers.filter((layer) => layer.type === LayerTypes.KML)).to.be.not.empty
+        })
+    })
+    it('edit a copy of drawing after page reload', () => {
+        cy.goToDrawing()
+        cy.clickDrawingTool(EditableFeatureTypes.ANNOTATION)
+        cy.get(olSelector).click('center')
+        cy.wait('@post-kml').then((interception) =>
+            cy.checkKMLRequest(interception, [EditableFeatureTypes.ANNOTATION])
+        )
+        cy.reload(true)
+        cy.waitUntilState((state) => {
+            return state.layers.activeLayers.length > 0
+        })
+        cy.openDrawingMode()
+        cy.wait('@get-kml')
+        cy.readWindowValue('drawingLayer')
+            .then((layer) => layer.getSource().getFeatures())
+            .should('have.length', 1)
```

Comment:
I'm not sure if this test thoroughly that you have a copy in your hands at this point.
Wouldn't it be a good thing to add another feature to the drawing, and check that the POST request is fired (instead of a PUT)?